testsuite/kafka.log

```diff
@@ -0,0 +1,1162 @@
+#
```

Comment:
This file shouldn't be in the git. It simplifies debugging that the output is dumped into root dir.
I'll add it to .gitignore file to prevent it from being automatically added for commit by IDEs.